### DISCUSSION TITLE
请问如何让机器人只在私聊中回复消息，而不屏蔽所有群的消息

### DISCUSSION THREAD

**Discussion by GeekBlackSheep** [nonebot/discussions#283] [问答]
以及如何让机器人指定在某个群或某个人中回复消息，其他好友和群的信息一律不做出回复

**Comment by yanyongyu** [comment]
自己编写一个rule白名单即可，或者使用`event preprocessor`过滤事件

**Reply by GeekBlackSheep** [reply]
好的，万分感谢，顺便能再问一下，rule和event preprocessor是写一个插件再导入，还是写在哪个配置文件里面？

**Reply by yanyongyu** [reply]
写插件里
